lib/spree/localized_number.rb

```diff
@@ -44,7 +44,7 @@ def localize_number(*attributes)
 
     def self.valid_localizable_number?(number)
       return true unless number.is_a?(String) || number.respond_to?(:to_d)
-      return false if number.to_s =~ /[.,]\d{2}[.,]/
+      return false if number.to_s =~ /[.,]\d{2}[.,]/ || number.to_s =~ /[^0-9,.]+/
```

Comment:
I like to include a comment to explain the intent of regular expressions (because they're seldom easy to understand!)

```suggestion
      # Invalid if only two digits between dividers, or if any non-number characters
      return false if number.to_s =~ /[.,]\d{2}[.,]/ || number.to_s =~ /[^0-9,.]+/
```

Reply:
(I'm choosing to ignore the fact that it strangely validates `1,23.00` as invalid,  but allows `1,2.00` and `1,2345.00`. Because `#parse` will handle it anyway)